src/api.ts

```diff
@@ -201,7 +211,14 @@ export interface TokenResponse {
   expires: number;
 }
 
-class ApiError extends Error {
+export interface LockOperationResult {
+  success: boolean;
+  message: string;
+  locked_by?: User | null;
+  lock_updated?: string | null;
```

Comment:
I recommend adding a comment saying what the string is (assuming it's not arbitrary); e.g. `// timestamp` or `// ISO 8601-formatted timestamp`